crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -308,9 +308,9 @@ impl CrossSigningResetHandle {
 pub enum CrossSigningResetAuthType {
     /// The homeserver requires user-interactive authentication.
     Uiaa(UiaaInfo),
-    /// OIDC is used for authentication and the user needs to open a URL to
+    /// OAuth 2.0 is used for authentication and the user needs to open a URL to
     /// approve the upload of cross-signing keys.
-    Oidc(OidcCrossSigningResetInfo),
+    Oauth(OauthCrossSigningResetInfo),
```

Comment:
I am wondering if we should remove the `Oidc`/`Oauth` prefix from this type since it is not namespaced in the Matrix spec after all, even if currently it is only used with OAuth 2.0 authentication.